unit_tests/utils/merkle/test_predicate.py

```diff
@@ -0,0 +1,30 @@
+from ethereum.utils import sha3
+
+from plasma_cash.utils.merkle.predicate import is_valid_proof
+
+
+class TestPredicate(object):
+
+    def test_proof_is_valid(self):
+        # Assume: leaves = {0: dummy_val, 1: dummy_val, 2: empty_val, 3: dummy_val}
+        empty_val = b'\x00' * 32
+        dummy_val = b'\x01' * 32
+        mid_left_val = sha3(dummy_val + dummy_val)
+        mid_right_val = sha3(empty_val + dummy_val)
+        root = sha3(mid_left_val + mid_right_val)
+        assert is_valid_proof(dummy_val, 0, dummy_val + mid_right_val, root) is True
```

Comment:
- Minor: doesn't need `is True`
- Minor: personal preference, could use 
```python
is_valid_proof(leaf=dummy_val, uid=0, proof=..., root=root)
```
 for better readability.